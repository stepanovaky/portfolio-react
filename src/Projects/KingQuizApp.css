.king-henry-quiz img {
    display: block;
    height: 200px;
    width: 100px;
    border: 2px solid rgb(75, 5, 75);
    border-radius: 5%;
    margin-left: auto;
    margin-right: auto;
}

.king-henry-quiz p {
    margin: 1em;
}