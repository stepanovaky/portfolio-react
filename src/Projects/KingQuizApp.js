import React from 'react';

function KingQuizApp() {
    return (
        <div className="king-henry-quiz" >
            <h2>King Henry VIII Quiz App</h2>
        </div>
    )
}

export default KingQuizApp;