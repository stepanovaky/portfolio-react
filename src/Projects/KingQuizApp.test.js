import React from 'react';
import renderer from 'react-test-renderer';
import KingQuizApp from './KingQuizApp';

it('renders correctly', () => {
    const tree = renderer
        .create(<KingQuizApp />)
        .toJSON();
    expect(tree).toMatchSnapshot();
})