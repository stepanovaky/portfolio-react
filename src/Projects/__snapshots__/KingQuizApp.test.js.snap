// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="king-henry-quiz"
>
  <h1>
    King Henry VIII Quiz App
  </h1>
</div>
`;
